### PR TITLE
feat(blog): add per page manifest blog post

### PR DESCRIPTION
**Pull request by Moocar**
blog post for the [per-page-manifest](https://github.com/gatsbyjs/gatsby/pull/14359#event-2402986461) changes that were recently merged to master

[Rendered View](https://github.com/Moocar/gatsby/blob/4e6c3d5c4f44972ad04741be4dac225b53c26fb5/docs/blog/2019-06-12-performance-improvements-for-large-sites/index.md)

### CODE REVIEW

File: docs/blog/2019-06-12-performance-improvements-for-large-sites/index.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+### Don't we have to wait on a network request on each navigation now?
+
+Prefetching FTW! Gatsby already prefetches any links on the page so that when the browser needs them, they're already in the cache. So when Gatsby makes a request for the `page-data.json`, it's already in the browser (assuming the user has been on the page long enough for the prefetches to finish). You can read more about this prefetching behavior in this blog[deep dive on Gatsby's performance optimizations](/blog/2019-04-02-behind-the-scenes-what-makes-gatsby-great/#gatsby-link-and-link-relprefetch).
+
+## Other benefits
+
+### Gatsby sites are now more "live"
+
+In Gatsby, all resources are content-hashed (except the html files). This includes the pages-manifest. So once a Gatsby site is loaded in the browser, the user will only ever see the resources generated during that build. If they stay on the site for days, they'll never see new content until they refresh.
```
Comment (KyleAMathews) [inline docs/blog/2019-06-12-performance-improvements-for-large-sites/index.md]: Should say "Previously in Gatsby" or something like that

File: docs/blog/author.yaml [hunk 2]
```diff
@@ -304,3 +304,7 @@
   bio: Writes things @ GatsbyJS. Chaotic good frontend dev, caffeine addict, backyard chicken wrangler 🐓.
   avatar: avatars/michelle-gienow.jpg
   twitter: "@michellegienow"
+- id: Anthony Marcar
+  bio: "todo: write this"
```
Comment (calcsam) [inline docs/blog/author.yaml]:
```suggestion
  bio: "Software consultant, Clojure enthusiast, and founder of Grabble (acquired by Walmart Labs)."
```

Comment (calcsam) [reply docs/blog/author.yaml]: ^ (if helpful ;)) 

listened to your podcast to see how you'd introduce yourself there and then you dodged the question 😂

Comment (Moocar) [reply docs/blog/author.yaml]: Only just saw this, you were very close to what I ended up writing (just replace bicycle tourer with founder ;) )

**Pull request merged**